client/index.html

```diff
@@ -40,6 +40,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-    <script type="module" src="src/index.tsx"></script>
+    <script type="module" src="./src/index.tsx"></script>
```

Comment:
Think this is problem with vite, if this wasn't included nested routes wouldn't work https://stackoverflow.com/questions/74312482/getting-404-not-found-when-refreshing-a-page-that-has-a-nested-route-because